tests/BenchmarkDotNet.Tests/Reports/SummaryTableTests.cs

```diff
@@ -59,6 +61,7 @@ public void NumericColumnIsRightJustified()
         public void TextColumnIsLeftJustified()
         {
             var config = ManualConfig.Create(DefaultConfig.Instance).AddColumn(new ParamColumn("Param"));
+            config.SummaryStyle = MockFactory.CreateSummaryStyle(textColumnJustification: SummaryTable.SummaryTableColumn.TextJustification.Left);
```

Comment:
Please revert these tests for the default values, and add new tests with override values.